gulpfile.js/import/importRoadmap.js

```diff
@@ -50,147 +50,174 @@ const TEXT_BLOCK_REGEX = /^#{1,3} (?:.(?!^#))*/gms;
+      // Escape amp-components in markdown to prevent them from being rendered as such
+      // plus remove Emojis and split into separate text blocks to allow smoother line breaks in frontend
       let body = issue.body.replace(AMP_COMPONENT_REGEX, ' `$1`');
-
-      // Remove Emojis and split into separate text blocks to allow smoother line breaks in frontend
       body = emojiStrip(body).trim().match(TEXT_BLOCK_REGEX);
 
       // Parse status update date from from issue title and set quarter
       let statusUpdate = title.match(STATUS_UPDATE_REGEX);
       let quarter;
       if (statusUpdate) {
-        quarter = `Q:${Math.ceil(parseInt(statusUpdate[2]) / 3)} ${
+        quarter = `Q${Math.ceil(parseInt(statusUpdate[2]) / 3)} ${
           statusUpdate[1]
         }`;
         statusUpdate = new Date(statusUpdate[0]).toDateString();
       } else {
         log.error(
-          `.. ${wg.name} - Failed for issue #${issue.number}: ${title}`
+          `.. ${workingGroupMeta.slug} - Could not parse valid date from issue title: ${title}`
         );
-        continue;
-      }
-
-      if (!workingGroups.includes(workingGroup)) {
-        workingGroups.push(workingGroup);
+        return;
       }
 
-      roadmap.push({
-        'wg_slug': workingGroupSlug,
-        'wg_name': meta.title,
+      return {
+        'wg_slug': workingGroupMeta.slug,
+        'wg_title': workingGroupMeta.title,
+        'wg_color': workingGroupMeta.color,
         'created_at': createdAt,
         'status_update': statusUpdate,
         'quarter': quarter,
         'number': issue.number,
         'html_url': issue.html_url,
         'body': body,
-      });
-    }
-
-    log.info(`.. ${wg.name} - ${issues.length} issues imported`);
-  }
-
-  // Get predefined color based on roadmap config
-  workingGroups = [...new Set(workingGroups.sort())];
-  workingGroups = workingGroups.map((group) => {
-    return {
-      'slug': group.slug,
-      'name': group.name,
-      'color': config.colors[group.slug] || config.fallbackColor,
-    };
-  });
-
-  // Sort roadmap entries, add matching working group color and evaluate
-  // what working groups have issued updates in which quarter
-  roadmap = roadmap.sort((a, b) => {
-    return new Date(b.status_update) - new Date(a.status_update);
-  });
-
-  const quarters = {'ordered': [], 'working_groups': {}};
-  for (const issue of roadmap) {
-    issue.color = config.colors[issue.wg_slug];
-    if (!quarters.ordered.includes(issue.quarter)) {
-      quarters.ordered.push(issue.quarter);
-    }
-    quarters.working_groups[issue.quarter] =
-      quarters.working_groups[issue.quarter] || [];
-    if (!quarters.working_groups[issue.quarter].includes(issue.wg_slug)) {
-      quarters.working_groups[issue.quarter].push(issue.wg_slug);
-    }
-  }
+      };
+    })
+    .filter((issue) => issue);
+}
 
+async function writeRoadmapYaml(roadmap) {
+  console.log('Write file');
```

Comment:
For debugging purposes? Otherwise use log helper and add a little creative writing 😉